boxsdk/object/task.py

```diff
@@ -0,0 +1,78 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import json
+
+from boxsdk.object.base_object import BaseObject
+from ..pagination.marker_based_object_collection import MarkerBasedObjectCollection
+
+
+class Task(BaseObject):
+    """Represents a Box task."""
+    _item_type = 'task'
+
+    def assign(self, assignee, assign_to_login=None):
```

Comment:
This should be split into two different methods; a user can't pass both params and expect something reasonable to happen